CHANGELOG.rst

```diff
@@ -11,6 +11,7 @@ Bug Fixes
 ---------
 * Fix incorrect metadata for compact counter tables (PYTHON-1100)
 * Call ConnectionException with correct kwargs (PYTHON-1117)
+* Fix "Can't connect to clusters built from source because version parsing doesn't handle 'x.y-SNAPSHOT'" (PYTHON-1118)
```

Comment:
`Can't connect to clusters built from source because version parsing doesn't handle 'x.y-SNAPSHOT' (PYTHON-1118)`

It's already in the Bug Fixes section, so no need to re-mention "Fix" IMO.